pkg/printers/internalversion/describe.go

```diff
@@ -1197,9 +1200,19 @@ func describePersistentVolume(pv *api.PersistentVolume, events *api.EventList) (
 	return tabbedString(func(out io.Writer) error {
 		w := NewPrefixWriter(out)
 		w.Write(LEVEL_0, "Name:\t%s\n", pv.Name)
-		printLabelsMultiline(w, "Labels", pv.Labels)
-		printAnnotationsMultiline(w, "Annotations", pv.Annotations)
-		w.Write(LEVEL_0, "Finalizers:\t%v\n", pv.ObjectMeta.Finalizers)
+		w.Write(LEVEL_0, "Namespace:\t%v\n", pv.ObjectMeta.Namespace)
+		if !pv.ObjectMeta.CreationTimestamp.IsZero() {
+			w.Write(LEVEL_0, "CreationTimestamp:\t%v\n", pv.ObjectMeta.CreationTimestamp.Time.Format(time.RFC1123Z))
+		}
+		if pv.ObjectMeta.DeletionTimestamp != nil {
```

Comment:
DeletionTimestamp is already printed at L1217. Maybe the ordering needs to change